### PR TITLE
Applied new table to existing pages.

### PR DESCRIPTION
**Pull request by joanofdart**
State issue number in your PR to close it when PR merged: issue #XXX
e.g. : "Resolve the bundle size ❤️: issue #1234"

### CODE REVIEW

File: libs/movie/src/lib/movie/pages/additional-information/additional-information.component.ts [hunk 1]
```diff
@@ -27,13 +27,34 @@ export class MovieFormAdditionalInformationComponent implements OnInit {
...
+  }
+
   constructor(private shell: MovieFormShellComponent) { }
 
   ngOnInit() {
     this.units$ = this.form.boxOffice.valueChanges.pipe(
       startWith(this.form.boxOffice.value),
       map((boxOffices) => boxOffices.map(({ unit }) => toUnit(unit)))
     );
+    console.log('form', this.form);
```
Comment (Mussuranae) [inline libs/movie/src/lib/movie/pages/additional-information/additional-information.component.ts:57]: Delete it, no ?

File: libs/movie/src/lib/movie/pages/artistic/artistic.component.html [hunk 2]
```diff
@@ -3,114 +3,100 @@ <h1 subtitle>Artistic Team</h1>
...
-        <span class="mat-caption">Add a Cast Member</span>
-      </footer>
+      <bf-form-table [columns]="castColumns" [form]="cast" buttonText="Add Cast Member">
+        <ng-template formView let-form>
+          <article [formGroup]="form" fxLayout="column" fxLayoutGap="24px">
+
+            <div fxLayout fxLayoutGap="24px">
+              <mat-form-field appearance="outline" fxFlex>
+                <mat-label>First Name</mat-label>
+                <input test-id="cast-first-name" matInput formControlName="firstName" required placeholder="eg: Cast Member’s First Name" />
```
Comment (Mussuranae) [inline libs/movie/src/lib/movie/pages/artistic/artistic.component.html:13]: Can you check the google spreadsheet for all placeholders, labels,etc ?

File: libs/movie/src/lib/movie/pages/artistic/artistic.component.ts [hunk 3]
```diff
@@ -12,6 +12,24 @@ export class MovieFormArtisticComponent {
   form = this.shell.form;
   public memberStatus = staticModels.MEMBER_STATUS;
 
+  public castColumns = {
+    firstName: 'First Name',
+    lastName: 'Last Name',
+    status: 'Status',
+    description: 'Description',
+    filmography: 'Filmography',
```
Comment (Mussuranae) [inline libs/movie/src/lib/movie/pages/artistic/artistic.component.ts:20]: the filmography is the table of 3 objects { title, year }, how is it display in the table ?

File: libs/movie/src/lib/movie/pages/artistic/artistic.component.html (hunk 2 above)
Comment (joanofdart) [reply libs/movie/src/lib/movie/pages/artistic/artistic.component.html:13]: Not in this PR, this is about adding the new component

File: libs/movie/src/lib/movie/pages/artistic/artistic.component.html [hunk 4]
```diff
@@ -3,114 +3,100 @@ <h1 subtitle>Artistic Team</h1>
...
-        </footer>
-      </article>
-      <footer fxLayout="column" fxLayoutAlign="center center">
-        <button mat-mini-fab color="primary" (click)="cast.add()" aria-label="Add a Cast Member">
-          <mat-icon svgIcon="add"></mat-icon>
-        </button>
-        <span class="mat-caption">Add a Cast Member</span>
-      </footer>
+      <bf-form-table [columns]="castColumns" [form]="cast" buttonText="Add Cast Member">
+        <ng-template formView let-form>
```
Comment (fritzschoff) [inline libs/movie/src/lib/movie/pages/artistic/artistic.component.html:7]: specify how filmography should be displayed

File: libs/movie/src/lib/movie/pages/artistic/artistic.component.html [hunk 5]
```diff
@@ -19,26 +20,39 @@ <h3>Cast Member</h3>
...
-                <mat-option *ngFor="let status of memberStatus" [value]="status.slug">
-                  {{ status.label }}
-                </mat-option>
-              </mat-select>
-            </mat-form-field>
+            <div fxLayout fxLayoutGap="24px">
+              <mat-form-field appearance="outline">
+                <mat-label>Status</mat-label>
+                <mat-select matInput formControlName="status">
+                  <mat-option *ngFor="let status of memberStatus" [value]="status.slug">
```
Comment (fritzschoff) [inline libs/movie/src/lib/movie/pages/artistic/artistic.component.html:27]: just use that static select here

**Pull request merged**